piccolo_api/fastapi/endpoints.py

```diff
@@ -121,7 +121,6 @@ def __init__(
 
         self.alias = f"{piccolo_crud.table._meta.tablename}__{id(self)}"
 
-        global ANNOTATIONS
```

Comment:
We need to keep this - we need to make sure the global ANNOTATIONS are updated. Is it causing mypy errors? If so, lets ignore the error.

Reply:
I put a `noqa` comment next to it.